modules/internal/Bytes.chpl

```diff
@@ -370,7 +305,7 @@ module Bytes {
     proc getByte(i: int): byteType {
```

Comment:
Perhaps this is a different issue, but isn't this routine supposed to be called `byte` to match `string`?

Reply:
I think it is a relic from the time where I had `iterBytes` and `getByte`. I don't think there is a big issue there. Good catch. I will change the name in this PR